src/modules/commander/state_machine_helper.cpp

```diff
@@ -669,20 +670,21 @@ bool set_nav_state(struct vehicle_status_s *status,
 		} else if (status_flags->data_link_lost_cmd) {
 			status->nav_state = vehicle_status_s::NAVIGATION_STATE_AUTO_RTGS;
 
-		} else if (status_flags->gps_failure_cmd) {
-			status->nav_state = vehicle_status_s::NAVIGATION_STATE_DESCEND;
-			enable_failsafe(status, old_failsafe, mavlink_log_pub, reason_no_gps_cmd);
-
 		} else if (status_flags->rc_signal_lost_cmd) {
 			status->nav_state = vehicle_status_s::NAVIGATION_STATE_AUTO_RCRECOVER;
 
 		} else if (status_flags->vtol_transition_failure_cmd) {
 			status->nav_state = vehicle_status_s::NAVIGATION_STATE_AUTO_RTL;
 
-			/* finished handling commands which have priority, now handle failures */
+		} else if (status_flags->gps_failure || status_flags->gps_failure_cmd) {
```

Comment:
what does the gps_failure_cmd exist and when is this set to true? 
I did not understand it from the flag description

Reply:
For testing presumably, but they aren't used by any test code we have. I left it in tact because I'm getting rid of all the cmds in the commander refactor https://github.com/PX4/Firmware/pull/7517.
For testing I'll use the actual gps_failure status flag instead.